app/dashboard/page.tsx

```diff
@@ -16,7 +15,7 @@ export default async function DashboardPage() {
   //   return null;
   // }
 
-  const dashboardData = await getDashboardMockDataAction("companyId");
+  // const dashboardData = await getDashboardMockDataAction("companyId");
```

Comment:
Remove these comments if not needed in the future (as each component would be responsible to grab its own data)